modules/accredited_representative_portal/db/migrate/20250128162530_create_ar_power_of_attorney_form_submissions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateArPowerOfAttorneyFormSubmissions < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ar_power_of_attorney_form_submissions do |t|
+      t.uuid :power_of_attorney_request_id, null: false
+      t.string :service_id
+      t.jsonb :service_response_ciphertext
+      t.string :status, default: 'pending'
```

Comment:
With this refined synchronization strategy, I believe the status values would need to be updated as well. Perhaps it is now these values and column needs to be NOT NULL?

enqueue_succeeded
enqueue_failed
succeeded
failed